src/article.html

```diff
@@ -200,17 +252,18 @@ <h1 class="text-primary text-center mb-4 title">Global temperature record shows
                 <a class="col-12 col-md mr-3 text-white" href="./empty.html?title=About">About</a>
                 <a class="col-12 col-md mr-3 text-white" href="./empty.html?title=Contact+us">Contact us</a>
                 <a class="col-12 col-md mr-3 text-white" href="./empty.html?title=Help">Help</a>
-                <a class="col-12 col-md mr-3 text-white" href="./empty.html?title=Terms+of+services">Terms of service</a>
+                <a class="col-12 col-md mr-3 text-white" href="./empty.html?title=Terms+of+services">Terms of
+                    service</a>
                 <a class="col-12 col-md mr-3 text-white" href="./empty.html?title=Privacy+policy">Privacy policy</a>
             </div>
         </div>
     </div>
-    
+
     <script type="text/javascript" src="./js/jquery.js"></script>
     <script type="text/javascript" src="./js/popper.js"></script>
     <script type="text/javascript" src="./js/bootstrap.js"></script>
     <script type="text/javascript" src="./js/common.js"></script>
     <script type="text/javascript" src="./js/article.js"></script>
 </body>
 
-</html>
+</html>
```

Comment:
Add new line to end of file. Some programs may have issues parsing lines not ending with newline chars.